Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM alpine
 
+ARG VERSION
```

Comment:
nitpick, maybe `FPM_VERSION` would be a better name, in case we need to configure the versions of other dependencies as well in the future